scripts/biomedical/VirtualMetabolicHuman/README.md

```diff
@@ -0,0 +1,136 @@
+
+The VHM database explicity connects human metabolism with genetics and human-associated microbial metabolism. It consists of five interconnected resources: `Human metabolism`, `Gut microbiome`, `Disease`, `Nutrition` and `ReconMaps`. More information on the database can be found on their official [website](https://www.vmh.life/#home) and [manuscript](https://academic.oup.com/nar/article/47/D1/D614/5146204).
+For the knowledge graph, three files are imported:
+
+- <u>recon-store-metabolites-1.tsv</u> : contains information on chemical properties of metabolites, and identifiers for various databases to query the metabolites. In order to generate dcids, the keggID for each metabolite was converted to the ChEMBL ID, using the [bioservices packages](https://bioservices.readthedocs.io/en/master/compound_tutorial.html). For the metabolites, with missing keggIDs, the chemical formulae were used to generate the dcids.
+- <u>recon-store-reactions-1.tsv</u> : contains information on the chemical equations and formulae of metabolic reactions, biochemical subsytem where the reactions take place and identifiers for various databases, to query the reactions. The reaction and metabolite abbreviations were mapped to one another using the longest substring match and the dcid for longest matched metabolite was appended to the respective reaction column. In addition, for generation of reaction dcids, the humanGEMIDs were obtained for reactions, using the metaNetX IDs and [Human1 data](https://raw.githubusercontent.com/khoahoang1891999/Importing-Human1-data/main/data/reactions.tsv). For reactions with no metaNetX IDs, the chemical abbreviations were used are reactions dcids.
+  In addition, this file was also used to generate a new file `subsystem.csv` which contains all the unique biochemical subsystems from the above reactions and the subsystem names were used to generate their corresponding dcids.ß
+- <u>recon-store-microbes-1.tsv</u>: contains information on microbial taxonomic classification, phenotype, metrics about genes, reactions and metabolites linked to microbes and identifiers for various databases, to query the microbes. The datacommons was queried for pre-exisiting microbe dcids using the scientific name, and for the ones with no node on datacommons, new dcids were generated using their genus and specie names. The dates were converted to ISO format.
+
+- ### Schema Overview
+
+The schema representing reaction, metabolite and microbiome data from VMH is defined in [VMH.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list.mcf) and [VMH_enum.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list_enum.mcf). The tmcfs for each of the corresponding csv files can be found [here](https://github.com/suhana13/data/tree/add_Virtual_metabolic_human_data/scripts/biomedical/VirtualMetabolicHuman/tmcf).
```

Comment:
schema will be stored in ChemicalCompund.mcf and ChemicalCompoundEnum.mcf

Reply:
Microbiome data will be stored in a separate mcf

Reply:
Done ! For the microbiome mcf, I'll change the link once the file's up there in schema dir, but for the rest of them, I've directed them to the chemical_compound.mcf and  chemical_compound_enum.mcf